project/EnsimeBuild.scala

```diff
@@ -286,6 +287,10 @@ object EnsimeTestingBuild {
     libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.1" % Test intransitive ()
   )
 
+  lazy val testingSimple2 = testingProject("testing/simple2") settings (
```

Comment:
is there any reason you need to create a new test project instead of re-using the existing one?